README.md

```diff
@@ -11,14 +11,17 @@ This app was originally developed as a [class project by U of T students](https:
 
 ### Via the front-end user-interface 
 
-When you [visit the app](https://trans-bdit.intra.prod-toronto.ca/traveltime-request/), you will be prompted to add/create at least one of each of the following:
-* a corridor, drawn on the map
-* a time range, given in hours of the day, 00 - 23
-* a date range (note that the end of the date range is exclusive)
-* a day of week selection
-* a selection of whether or not to include statutory holidays 
-
-The app will combine these factors together to request travel times for all possible combinations. If one of each type of factor is selected, only a single travel time will be estimated with the given parameters. 
+When you [visit the app](https://trans-bdit.intra.prod-toronto.ca/traveltime-request/), you will be prompted to create at least one of each of the following factors:
+
+| Factor | Description |
+| ----- | ----------- |
+| Corridor | Drawn on the map, it is a shortest path between two intersections of your choice. Draw it in both directions if you need both directions of travel. |
+| Time Range | Times must start and end on the hour and the app accepts integer values between 0 and 24. The final hour is _exclusive_, meaning that a range of 7am to 9am covers two hours, not three. Values of 0 and 24 both interchangeably represent midnight; a time range of 0 - 24 will return all hours of the day. A time range starting after it ends (e.g. 10pm to 4am) will wrap around midnight. |
+| Date Range | Use the calendar widget to select a date range. Note that selected ranges are displayed with an exclusive end date. |
```

Comment:
Is it worth discussing what happens when you wrap around midnight? If you have a range of [day, day+1), the next day is not included. What happens when you input for your time, say [22, 5)? This may be confusing for the reader.